### PR TITLE
feat: update events status

### PR DESCRIPTION
**Pull request by vivianamck**
## Issue ticket number and link

https://trello.com/c/gisYIZSn

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [X] :sparkles: Feature
- [ ] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

- Events list with options edit and remove

## What is the new behavior?

- Events list with new option of status change

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: src/components/EventsAdminView/EventAdminView.tsx [hunk 1]
```diff
@@ -116,12 +139,28 @@ export default function EventAdminView({
...
+    status: ConferenceStatus
+  ) => {
+    const data = await api.update(id, { status })
+    if (data.status === 200) updateStatusEvents(id, status)
+  }
+
+  const updateStatus = (event: Conference) => {
+    event.status !== 'active'
+      ? updateEvent(event._id, 'active')
+      : updateEvent(event._id, 'inactive')
```
Comment (jlora-mck) [inline src/components/EventsAdminView/EventAdminView.tsx:159]: Please refactor this so we dont repeat yourself

File: src/components/EventsAdminView/EventAdminView.tsx [hunk 2]
```diff
@@ -116,12 +139,28 @@ export default function EventAdminView({
...
+  }
+
+  const validateStatus = (status: ConferenceStatus) => status !== 'active'
```
Comment (jlora-mck) [inline src/components/EventsAdminView/EventAdminView.tsx:162]: Now that we are using a lot of active and another statuses , please create an enum for this

File: src/components/EventsAdminView/EventAdminView.tsx [hunk 3]
```diff
@@ -116,12 +139,28 @@ export default function EventAdminView({
...
 
     setButton(false)
   }
 
+  const updateEvent = async (
+    id: string | undefined,
+    status: ConferenceStatus
+  ) => {
+    const data = await api.update(id, { status })
+    if (data.status === 200) updateStatusEvents(id, status)
```
[the remark this answers is not in the document]
Comment (vivianamck) [reply src/components/EventsAdminView/EventAdminView.tsx:153]: For update service back and information of front

File: src/components/EventsAdminView/EventAdminView.tsx (hunk 1 above)
Comment (vivianamck) [reply src/components/EventsAdminView/EventAdminView.tsx]: Done

**Review by jlora-mck — APPROVED** [review]

**Review by FelipeLosada1996 — APPROVED** [review]

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**